src/flow.h

```diff
@@ -76,6 +77,8 @@ typedef struct AppLayerParserState_ AppLayerParserState;
 #define FLOW_TOCLIENT_DROP_LOGGED         0x00004000
 /** alproto detect done.  Right now we need it only for udp */
 #define FLOW_ALPROTO_DETECT_DONE          0x00008000
+/** time machine is enabled on this flow */
+#define FLOW_TIMEMACHINE_ENABLED          0x00010000
```

Comment:
Time machine enabled means that we're not adding packets to the TM as we're already in the process of dumping them to disk as we receive them?